code/modules/projectiles/projectile/bullets.dm

```diff
@@ -53,6 +53,18 @@
 /obj/item/projectile/bullet/heavybullet
 	damage = 35
 
+/obj/item/projectile/bullet/sabot
+	damage = 150
+	var/breakthings = FALSE
+	armour_penetration = -5 //NOT A TYPO
+
+/obj/item/projectile/bullet/sabot/Range()
+	..()
+	damage += 70

```

Comment:
Actually, on inspection it'll do 220 damage to someone literally right next to you. That's... probably bad.